client/components/LearnerViewDirective.js

```diff
@@ -733,6 +739,27 @@ tie.directive('learnerView', [function() {
         };
 
         /**
+          * Highlights syntax errors in the coding UI
+          *
+          * @param {number} lineNumber
+          */
+        var highlightLine = function(lineNumber) {
+          var actualLineNumber = lineNumber - 1;
+          var codeLines = document.querySelectorAll('.CodeMirror-line');
+          codeLines[actualLineNumber].classList.add('tie-syntax-error-line');
```

Comment:
Pull .tie-syntax-error-line out into a named constant since you use it multiple times